lib/Qi.pm

```diff
@@ -10,7 +14,9 @@ sub startup {
   my $config = $self->plugin('Config');
 
   #Model
-   $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(sql => sub{ state $sql = SQL::Abstract->new});
```

Comment:
My thought process was that if I put it in the startup, it would only be created once and available forever, but if I put it in the model, it would need to be created every time the model was called. Unfortunately, I'm not familiar enough to know if my thought process has any merit